pkg/controller/manager/manager_controller_test.go

```diff
@@ -200,18 +196,14 @@ var _ = Describe("Manager controller tests", func() {
 		})
 
 		It("should reconcile if user supplied a manager TLS cert", func() {
-			// Create a manager cert secret with invalid DNS name
+			// Create a manager cert secret.
 			dnsNames := []string{"manager.example.com", "192.168.10.22"}
+			testCA := test.MakeTestCA("manager-test")
 			secret, err := render.CreateOperatorTLSSecret(
-				nil, render.ManagerTLSSecretName, render.ManagerSecretKeyName, render.ManagerSecretCertName, render.DefaultCertificateDuration, nil, dnsNames...)
+				testCA, render.ManagerTLSSecretName, render.ManagerSecretKeyName, render.ManagerSecretCertName, render.DefaultCertificateDuration, nil, dnsNames...)
```

Comment:
Can you add a test to this down below to verify that the secret created here matches the one after the Render?
Maybe we should even include a second Reconcile call here because that would have caught the previous problem.

Reply:
A second Reconcile would really only be to specifically catch if we put on owner references and then used them for something so a 2nd reconcile won't do much. So I wouldn't add a 2nd.